docs/user_profiles.md

```diff
@@ -0,0 +1,22 @@
+# Managing user profiles
```

Comment:
can we rename this file to `odh_user_profiles.md`?

Reply:
or nvm this includes opf stuff like ceph buckets so a general `user_profiles.md` works too